### PR TITLE
Visual Campaign Central Improvements

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
1. Switch path and device tabs
2. Provide more logical groupings for device breakdown
3. Cleanup some unnecessary code.

### CODE REVIEW

File: app/model/LatestAnalyticsBreakdownItem.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  implicit val latestCampaignAnalyticsBreakdownItemFormat: Format[LatestAnalyticsBreakdownItem] =
+    Jsonx.formatCaseClass[LatestAnalyticsBreakdownItem]
+
+  implicit val latestAnalyticsMonoid: Monoid[LatestAnalyticsBreakdownItem] =
+    new Monoid[LatestAnalyticsBreakdownItem] {
+      override def combine(itemOne: LatestAnalyticsBreakdownItem,
+                           itemTwo: LatestAnalyticsBreakdownItem): LatestAnalyticsBreakdownItem = {
+        LatestAnalyticsBreakdownItem(
+          uniques = itemOne.uniques + itemTwo.uniques,
```
Comment (janua) [inline app/model/LatestAnalyticsBreakdownItem.scala:26]: What context is this addition done in? Uniques usually can't be added together unless it's over the same range + more days?

**Comment by janua** [comment]
:+1:

**Review by kelvin-chappell — APPROVED** [review]
👍

File: app/model/LatestAnalyticsBreakdownItem.scala (hunk 1 above)
Comment (LATaylor-guardian) [reply app/model/LatestAnalyticsBreakdownItem.scala:26]: We discussed and it's safe we believe in this context. But context is important.

**Pull request merged**